src/EditorFeatures/CSharpTest/Diagnostics/FixReturnType/FixReturnTypeTests.cs

```diff
@@ -0,0 +1,183 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.CodeFixes.FixReturnType;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics.FixReturnType
+{
+    [Trait(Traits.Feature, Traits.Features.CodeActionsFixReturnType)]
+    public partial class FixReturnTypeTests : AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest
+    {
+        internal override (DiagnosticAnalyzer, CodeFixProvider) CreateDiagnosticProviderAndFixer(Workspace workspace)
+            => (null, new CSharpFixReturnTypeCodeFixProvider());
+
+        [Fact]
+        public async Task Simple()
+        {
+            await TestInRegularAndScript1Async(
+@"class C
+{
+    void M()
+    {
+        [|return|] 1;
+    }
+}",
+@"class C
+{
+    int M()
+    {
+        return 1;
+    }
+}");
+        }
+
+        [Fact]
+        public async Task ReturnString()
+        {
+            await TestInRegularAndScript1Async(
+@"class C
+{
+    void M()
+    {
+        [|return|] """";
+    }
+}",
+@"class C
+{
+    string M()
+    {
+        return """";
+    }
+}");
+        }
+
+        [Fact]
+        public async Task ReturnNull()
+        {
+            await TestMissingInRegularAndScriptAsync(
+@"class C
+{
+    void M()
+    {
+        [|return|] null;
```

Comment:
perhaps offer to just be 'object'?  we often infer that as the 'default' type to use if we can't figure out something better.